src/ucs/sys/sys.c

```diff
@@ -932,7 +933,10 @@ double ucs_get_cpuinfo_clock_freq(const char *header, double scale)
     fclose(f);
 
     if (warn) {
-        ucs_warn("Conflicting CPU frequencies detected, using: %.2f", value);
+        level = ucs_global_opts.warn_inv_tsc ? UCS_LOG_LEVEL_WARN :
+                                               UCS_LOG_LEVEL_DEBUG;
+        ucs_log(level,
```

Comment:
maybe do some wrapper for the logging and checking warn_inv_tsc , e.g ucs_warn_inv_tsc() macro
it's used 3 times in the code